modifications/update-console-sources

```diff
@@ -124,7 +124,7 @@ docker cp . $container:/opt/app-root/src/
 run="docker exec $container bash -c"
 $run "yum install -y python3"
 $run "tar fx yarn-offline.tar"
-$run "npm install ./yarn-1.9.4.tgz"
+$run "npm install ./yarn-1.22.10.tgz"
```

Comment:
```suggestion
$run "npm install ./yarn-1.9.4.tgz"
```

Reply:
no, the point of this is to update both nodejs and yarn

Reply:
new nodejs-headers and yarn tarballs will of course have to be uploaded to lookaside cache

Reply:
we decided to try w/o updating yarn but anyway this is obsolete